### DISCUSSION TITLE
Administrative Boundary Polygon is missing (Spain, Relation 1311341)

### DISCUSSION THREAD

**Discussion by j-bbr** [osm2pgsql-dev/osm2pgsql#1761] [Q&A] [answered]
Hi,
I am working with administrative boundaries and I found out that in both the Europe and Spain osm pbf Extract from Geofabrik the polygon for Spain is missing.  All other countries boundaries are fine. I am importing with the following flags
`osm2pgsql -s -d 'CONNECTION STRING' -c spain-latest.osm.pbf -E 4326 --hstore-all`
#866 There was some discussion about this a while back so I was wondering whether that is a data problem or with osm2pgsql.

Version:

2022-09-14 15:06:37  osm2pgsql version 1.7.0
Build: Release
Compiled using the following library versions:
Libosmium 2.17.3
Proj [API 6] 9.1.0
Lua 5.1.4 (LuaJIT 2.1.0-beta3)

**Comment by lonvia** [comment, marked as answer]
There is an issue with the Europe/Spain extract from Geofabrik (which @woodpeck hopefully is already aware of). It does not contain the full Spain boundary anymore.

**Reply by lonvia** [reply]
To be precise: https://www.openstreetmap.org/way/114297923 is missing while https://www.openstreetmap.org/way/440451928 is included in the extracts. That makes an open ring around that naval base.

**Reply by j-bbr** [reply]
Unfortunately it's not just missing in the Spain extract but also in the complete Europe extract but it's not an OSM or osm2pgsql problem then. 
Edit: Sorry I just reread and understood that you meant both Europe and Spain extract
